Simulator/loot_orb.py

```diff
@@ -0,0 +1,148 @@
+from Simulator.item_stats import item_builds as full_items, starting_items
+
+item_list = list(full_items.keys())
+
+# Loot Orbs
+# There are 3 types of orbs:
+# Common (Gray): 3 gold worth of champions, gold, or champion duplicator
+# Uncommon (Blue): 6 gold worth of champions, gold, or items
+# Rare (Gold): 10 gold worth of champions, gold, or spatula
+# TODO add reforgers, magnetic removers, emblem tomes, and item choosers
+
+class LootOrb(Enum):
+    COMMON = {
+        'three_gold': .2,
+        'three_cost': .25,
+        'two_cost_one_gold': .25,
+        'three_one_costs': .25,
+        'champion_duplicator_one_gold': .05
+    }
+    UNCOMMON = {
+        'six_gold': .05,
+        'two_three_costs': .15,
+        'three_two_costs': .15,
+        'one_item': .6,
+        'champion_duplicator_one_gold_two_cost': .05
+    }
+    RARE = {
+        'ten_gold': .15,
+        # 'two_five_costs': .1,
+        'champion_duplicator_five_gold': .1,
+        'spatula': .75,
+    }
+
+# Implementation of the loot that can be given to the player
+# The rewards are pretty self explanatory; 'three_one_costs' gives three random one cost champions, 'one_item' gives one random item, etc.
+def give_loot(player, reward):
+    # I would use a match here but we're on python 3.8
+    if reward == 'three_gold':
+        player.gold += 3
+    if reward == 'six_gold':
+        player.gold += 6
+    if reward == 'ten_gold':
+        player.gold += 10
+    if reward == 'three_one_costs':
+        give_champions(player, pool_stats.COST_1, count=3)
+    if reward == 'two_cost_one_gold':
+        give_champions(player, pool_stats.COST_2)
+        player.gold += 1
+    if reward == 'three_cost':
+        give_champions(player, pool_stats.COST_3)
+    if reward == 'three_two_costs':
+        give_champions(player, pool_stats.COST_2, count=3)
+    if reward == 'two_three_costs':
+        give_champions(player, pool_stats.COST_3, count=2)
+    if reward == 'two_five_costs':
+        give_champions(player, pool_stats.COST_5, count=2)
+    if reward == 'one_item':
+        give_random_item(player)
+    if reward == 'full_item':
+        give_random_full_item(player)
+    if reward == 'champion_duplicator_one_gold':
+        player.gold += 1
+        player.add_to_item_bench('champion_duplicator')
+    if reward == 'champion_duplicator_one_gold_two_cost':
+        player.gold += 1
+        give_champions(player, pool_stats.COST_2)
+        player.add_to_item_bench('champion_duplicator')
+    if reward == 'champion_duplicator_five_gold':
+        player.gold += 5
+        player.add_to_item_bench('champion_duplicator')
+    if reward == 'spatula':
+        player.add_to_item_bench('spatula')
+
+
+# Utility Functions
+
+## Random Chapmions
+def give_champions(player, cost, count=1):
+    for _ in range(count):
+        give_champion(player, cost)
+
+def give_champion(player, cost):
+    # Give gold if bench is full
+    if player.bench_full():
```

Comment:
If I'm not mistaken, this section of code is actually already handled in the add_to_bench. If there is no room on the bench for the champion, it gets automatically sold. Probably can save a few lines of code here.